source/gui/src/parametermonitorform.cpp

```diff
@@ -44,6 +44,10 @@ ParameterMonitorForm::ParameterMonitorForm(QWidget *parent) :
         {
             emit biasEnableChanged(checked);
         } );
+    connect(ui->hiGainCheckBox, &QCheckBox::clicked, this, [this](bool checked)
+        {
+            emit gainSwitchChanged(checked);
+        } );
```

Comment:
Signals can be connected to other signals for automatic forwarding. No lambda necessary.
```suggestion
    connect(ui->hiGainCheckBox, &QCheckBox::clicked, this, &ParameterMointorForm::gainSwitchChanged);
```